### DISCUSSION TITLE
Swing Music - like Spotify?

### DISCUSSION THREAD

**Discussion by jcolinger3** [swingmx/swingmusic#167] [Q&A] [answered]
I'm confused... does Swing Music create curated/automated playlists based on a search/channel/genre?  Or am I still required to create playlists or folders to do that?  
The feature of Spotify, Pandora, other streaming services that I enjoy a lot is a "set it and forget it" type of listening experience.  It seems like Swing still requires the playlist/folder method.
Can someone explain?  Otherwise, this is nothing like Spotify... other than it is streaming music.

**Comment by ThinkChaos** [comment]
My understanding is the "like a cooler Spotify" refers mostly to the design.

**Reply by jcolinger3** [reply]
Oh. "cooler Spotify" is just the design/UI?  But just a music player that doesn't do anything special.

**Reply by ThinkChaos** [reply]
Well cooler because it's your own files and server, and Spotify because of the UI (edit:) and streaming.  
I could be wrong, that's just my understanding from reading it as a third party..

**Reply by jcolinger3** [reply, marked as answer]
[What makes Spotify cool](https://www.google.com/search?q=what+makes+spotify+cool&oq=what+makes+spotify+cool&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQIRigATIHCAIQIRifBTIHCAMQIRifBTIHCAQQIRifBTIHCAUQIRifBdIBCDUxNzZqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8&safe=active&ssui=on)

**Reply by ThinkChaos** [reply]
I personally don't care podcasts, social, auto curating, etc. Self hosting and owning my music and data is "cooler" for me.  
I guess if you want "set it and forget it" you're just not going to be into that and it's fine, but no need to make it seems like this is  any less good when it's just your personal goals don't align.

**Reply by jcolinger3** [reply]
Very well then.  Difference of opinion.  
Agree to disagree.
